[
  {
    "project": "bdd100k/bdd100k",
    "commit": "d2e5b81ec7eda5b28c7f429e48749d0a4fb87fa7",
    "filename": "bdd100k/eval/seg.py",
    "min_patch_found": false,
    "full_warning_msg": "bdd100k/eval/seg.py:93:41 Incompatible parameter type [6]: Expected `typing.Iterable[Variable[_T2]]` for 2nd positional only parameter to call `zip.__new__` but got `int`.",
    "exception": "PatchSet should only have 1 Patch, but it has 0",
    "dd_fail": true
  },
  {
    "project": "bdd100k/bdd100k",
    "commit": "d2e5b81ec7eda5b28c7f429e48749d0a4fb87fa7",
    "filename": "bdd100k/label/to_mask.py",
    "filename_after_commit": "/home/chowyi/TypeAnnotation_Study/GitHub/bdd100k-bdd100k/bdd100k/label/to_mask.py",
    "file_hunks_size": 11,
    "min_patch_found": false,
    "single_hunk": false,
    "fit_TFix": false,
    "delete_only_patch": false,
    "has_suppression_all_hunks": false,
    "full_warning_msg": "bdd100k/label/to_mask.py:435:4 Call error [29]: `Variable[_VT]` is not a function.",
    "message": " `Variable[_VT]` is not a function.",
    "rule_id": "Call error [29]",
    "warning_line_no": 435,
    "warning_line": "    convert_funcs[args.mode]("
  }
]